src/components/Products/Products.js

```diff
@@ -0,0 +1,161 @@
+import React from 'react';
+import TablePagination from '@mui/material/TablePagination';
+import {
+    TableContainer,
+    Table,
+    TableHead,
+    TableBody,
+    TableRow,
+    TableCell,
+    Paper,
+    Button
+} from '@mui/material';
+import DeleteIcon from '@mui/icons-material/Delete';
+import EditIcon from '@mui/icons-material/Edit';
+import { useGetProductsQuery, useAddProductMutation, useUpdateProductMutation, useDeleteProductMutation } from '../../services/product';
```

Comment:
Add baseurl in jsconfig  so that te imports can be absolute
https://www.bezkoder.com/absolute-import-react/